### DISCUSSION TITLE
Out-of-the-box backend: Add Supabase and MailerSend to the starter kit

### DISCUSSION THREAD

**Discussion by sbutler-gh** [navneetsharmaui/sveltekit-starter#6] [Ideas]
Here is the Supabase library from the Svelte Add collection:

https://github.com/joshnuss/svelte-supabase

MailerSend is an open-source transactional email service:

https://www.mailersend.com/

Both of these services have been fluid to integrate in my experience, and if included in a starterkit, it gives the basis for a full-stack app out-of-the-box (Supabase handles authentication emails automatically)

**Comment by navneetsharmaui** [comment]
@sbutler-gh Thar could be done.
I'm planning on such advanced features for the starter kit through a node script i.e. when users clone or download this repo they can run that script to add additional advance features like that which normally most people don't need.
